api/csv_generator.py

```diff
@@ -0,0 +1,78 @@
+import random
+import pandas as pd
+from faker import Faker
+from datetime import datetime, timedelta
```

Comment:
I believe standard practice in Python is to define __init__.py and place the imports there, but I may be mistaken. Please double check.